### PR TITLE
feature/TT-2552 Handle Initial User

### PR DESCRIPTION
**Pull request by gtryus**
- Work offline button for team shown when appropriate
- User list adjusted based on chosen button
- update localization

### CODE REVIEW

File: src/routes/Welcome.tsx [hunk 1]
```diff
@@ -151,22 +169,36 @@ export function Welcome(props: IProps) {
...
   };
 
   const handleGoOffline = () => {
     handleOfflineChange('offline');
   };
+
   const handleQuickOnline = () => {
-    localStorage.setItem('autoaddProject', 'true');
-    handleGoOnline();
+    if (!hasOfflineProjects) localStorage.setItem('autoaddProject', 'true');
```
Comment (sarahentzel) [inline src/routes/Welcome.tsx:185]: why are we checking for offline projects when the user is going online?

File: src/routes/Welcome.tsx [hunk 2]
```diff
@@ -179,14 +211,13 @@ export function Welcome(props: IProps) {
...
-      AddRecord(t, userRec, user, memory)
-    );
+    await AddUserLocalOnly(userRec);
     await offlineSetup();
     return userRec.id;
   };
+
   const handleQuickOffline = () => {
-    localStorage.setItem('autoaddProject', 'true');
+    if (!hasOfflineUsers) localStorage.setItem('autoaddProject', 'true');
```
Comment (sarahentzel) [inline src/routes/Welcome.tsx:220]: why aren't we checking offline project here instead of users?

Comment (gtryus) [reply src/routes/Welcome.tsx:220]: On this one, the issue is that if you have an offline user, you have likely already created an offline project so you don't automatically want to create a new one each time you come back to the welcome screen and choose this option.

Comment (gtryus) [reply src/routes/Welcome.tsx:220]: I think checking offline projects might do the same thing and may help us with the case where the user doesn't actually create a project the first time they click this option...

File: src/routes/Welcome.tsx (hunk 1 above)
Comment (gtryus) [reply src/routes/Welcome.tsx:185]: I guess in both these cases we may just want to check if they have any projects in their local cache. I'll make these changes.

File: src/routes/Welcome.tsx (hunk 2 above)
Comment (sarahentzel) [reply src/routes/Welcome.tsx:220]: but shouldn't it check online projects then for go online?

File: src/routes/Welcome.tsx (hunk 1 above)
Comment (sarahentzel) [reply src/routes/Welcome.tsx:185]: but shouldn't it check online projects then for go online?

**Pull request merged**